build/SharedFx.props

```diff
@@ -0,0 +1,44 @@
+<Project>
+  <PropertyGroup>
+    <!-- directories -->
+    <_WorkRoot>$(RepositoryRoot).w\</_WorkRoot>
+    <_WorkLayoutDir>$(_WorkRoot).l\</_WorkLayoutDir>
+    <_WorkOutputDir>$(_WorkRoot).o\</_WorkOutputDir>
+    <_SrcDir>$(RepositoryRoot)src\</_SrcDir>
+    <_AllMetapackageDir>$(_SrcDir)Microsoft.AspNetCore.All\</_AllMetapackageDir>
+    <_TemplatesDir>$(MSBuildThisFileDirectory)tools\templates\</_TemplatesDir>
+    <_DockerDir>$(MSBuildThisFileDirectory)tools\docker\</_DockerDir>
+    <_PackagingDir>$(MSBuildThisFileDirectory)tools\packaging\</_PackagingDir>
+    <_DebToolDir>$(MSBuildThisFileDirectory)tools\dotnet-deb-tool-consumer\</_DebToolDir>
+    <_SharedFxSourceDir>$(RepositoryRoot).deps\Signed\SharedFx\</_SharedFxSourceDir>
+    <_InstallerSourceDir>$(RepositoryRoot).deps\Installers\</_InstallerSourceDir>
+    <_DockerRootDir>/opt/code/</_DockerRootDir>
+    <_InstallersOutputDir>$(ArtifactsDir)installers\</_InstallersOutputDir>
+
+    <!-- installers -->
+    <SharedFxInstallerName>aspnetcore-shared</SharedFxInstallerName>
+    <HostingInstallerName>dotnet-hosting</HostingInstallerName>
+    <DebConfigInFile>$(_PackagingDir)debian_config.json.in</DebConfigInFile>
+
+    <PublicCoreFeedPrefix>https://dotnetcli.blob.core.windows.net/dotnet</PublicCoreFeedPrefix>
+    <DotNetAssetRootUrl Condition="'$(DotNetAssetRootUrl)'==''">$(PublicCoreFeedPrefix)</DotNetAssetRootUrl>
+
+    <DotnetRuntimePackageName>dotnet-runtime</DotnetRuntimePackageName>
+    <DotnetRuntimeInstallerName>$(DotnetRuntimePackageName)-$(MicrosoftNETCoreApp21PackageVersion)</DotnetRuntimeInstallerName>
+    <DotnetRuntimeInstallerArchiveName>$(DotnetRuntimeInstallerName)-linux-x64.tar.gz</DotnetRuntimeInstallerArchiveName>
+    <RuntimeTargzLink>$(DotNetAssetRootUrl)/Runtime/$(MicrosoftNETCoreApp21PackageVersion)/$(DotnetRuntimeInstallerArchiveName)</RuntimeTargzLink>
+    <SharedFxLinuxArchiveFilePath>$(_SharedFxSourceDir)$(SharedFxInstallerName)-$(PackageVersion)-linux-x64.tar.gz</SharedFxLinuxArchiveFilePath>
+
+    <!-- installer metadata -->
+    <MaintainerName>Microsoft</MaintainerName>
+    <MaintainerEmail>nugetaspnet@microsoft.com</MaintainerEmail>
+    <Homepage>https://www.asp.net/</Homepage>
+    <InstallRoot>/usr/share/dotnet</InstallRoot>
+    <PackageRevision>1</PackageRevision>
```

Comment:
When (if ever) would this get changed?

Reply:
We will never change this really given the way we version installers. I've kept it as a property since this value is shared for both RPM and DEB installers. Could hardcode but I don't think that's necessary.